src/matrixtypes.ts

```diff
@@ -42,6 +42,7 @@ export interface IMatrixEvent {
     unsigned?: any; // tslint:disable-line no-any
     origin_server_ts?: number;
     users?: any; // tslint:disable-line no-any
+    users_default?: any; // tslint:disable-line no-any
```

Comment:
hol up,

isn't this part of the event content?

Reply:
yeah, which is why it goes here. A matrix event (in this case, the `m.permissions` event) *can* have that as its content

Reply:
ah